library/connect-go/v0.4.0/buf.plugin.yaml

```diff
@@ -4,12 +4,12 @@ plugin_version: v0.4.0
 source_url: github.com/bufbuild/connect-go
 description: A better gRPC.
 deps:
-  - plugin: buf.build/library/go:v1.28.0
+  - plugin: buf.build/library/go:v1.28.1
```

Comment:
Bumped this to the latest version (upstream connect-go started updating this dependency w/ dependabot).